spec/02-integration/04-admin_api/06-certificates_routes_spec.lua

```diff
@@ -3,7 +3,7 @@ local dao_helpers = require "spec.02-integration.03-dao.helpers"
 local DAOFactory = require "kong.dao.factory"
 local helpers = require "spec.helpers"
 local cjson = require "cjson"
-
+local utils = require "kong.tools.utils"
```

Comment:
We are now missing a blank line between the required modules and the below helper function (there was two before this line was changed to another `require`).